src/api/auth.c

```diff
@@ -151,7 +152,22 @@ int check_client_auth(struct ftl_conn *api, const bool is_api)
 			// Mark SID as available
 			sid_avail = true;
 		}
+	}
 
+	// If not, does the client provide a session ID via URI?
+	if(!sid_avail && api->request->query_string && GET_VAR("sid", sid, api->request->query_string) > 0)
+	{
+		// "+" may have been replaced by " ", undo this here
```

Comment:
Where should such a conversion happen? (`+` -> ` `)

Reply:
https://github.com/pi-hole/FTL/blob/a5d0a663e40ed12d0d5ee608ff2523cd5208a908/src/civetweb/civetweb.c#L14096-L14101

We typically want decoding to happen, so we cannot simply disable this